arctic/arctic.py

```diff
@@ -144,12 +144,14 @@ def _conn(self):
             return self.__conn
 
     def reset(self):
+        logger.debug("Arctic.reset()")
```

Comment:
can't tell if this is an actual issue or a github formatting issue, but this line looks to be indented incorrectly.

Reply:
Looks ok to me...

Reply:
ok, on my monitoir it had one space more than the line below it. :)